src/bp/ui-admin/src/Pages/Confusion/gradient.js

```diff
@@ -0,0 +1,41 @@
+import _ from 'lodash'
+
+const gradient = {
```

Comment:
this will eventually (soon) be used in studio as well, we might want to make this a share component